packages/utils/src/date/index.ts

```diff
@@ -175,10 +179,45 @@ export function timeZoneExists(timeZone: string): boolean {
 	}
 }
 
+/**
+ * Date format options
+ * @enum string
+ */
+export const FORMAT = {
+	/** en: June 29, 2021 / fr: 29 juin 2020 / ja: 2020年6月29日 */
+	MDY_LONG: 'MDY_LONG',
+	/** en: June 2020 / fr: juin 2020 / ja: 2020年6月 */
+	MY_LONG: 'MY_LONG',
+	/** en: 06/29/2020 / fr: 29/06/2020 / ja: 2020/06/29 */
+	MDY: 'MDY',
+	/** en: 6/29/20, 10:00 PM / fr: 29/06/2020 22:00 / ja: 2020/06/29 22:00 */
+	MDYHM: 'MDYHM',
+};
+
+const options: Map = {
+	[FORMAT.MDY_LONG]: { year: 'numeric', month: 'long', day: 'numeric' },
+	[FORMAT.MY_LONG]: { year: 'numeric', month: 'long' },
+	[FORMAT.MDY]: { year: 'numeric', month: '2-digit', day: '2-digit' },
+	[FORMAT.MDYHM]: { dateStyle: 'short', timeStyle: 'short' },
+};
+
+/**
+ * Format a date using Intl.
+ * @param date {DateFnsFormatInput} A date: Date, string or Number
+ * @param dateOption {string} Comes from FORMAT enum
+ * @param lang {string} language
+ * @returns The formated date
+ */
+export const format = (date: DateFnsFormatInput, dateOption: string, lang: string): string => {
+	return new Intl.DateTimeFormat(lang, options[dateOption]).format(parse(date));
```

Comment:
```suggestion
export const format = (date: DateFnsFormatInput, options: DateTimeFormatOptions, lang: string): string => {
	return new Intl.DateTimeFormat(lang, options).format(parse(date));
```
Would be simplier and allow customisation.. or at least you should type "dateOption" to instance of FORMAT

Reply:
The goal here was to limit the use of this formating method to standard formats and not to allow multiple ones.
Of course the list of format will evolve depending of the application needs.
But if we let the user use any format he wants, it's open and allow customisation, but it will be more difficult if one day we want to migrate to another formating API.